modules/installation-mirror-repository.adoc

```diff
@@ -76,11 +113,84 @@ imageContentSources:
   - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
 ...
+endif::install[]
+ifdef::update[]
+To use the new mirrored repository for upgrades, use the following to create an ImageContentSourcePolicy:
+
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+...
```

Comment:
Nit, you could have this ellipsis land outside the `endif::update[]` and be shared between the install and update cases.